cli/dev/hznenv.go

```diff
@@ -1,33 +1,19 @@
 package dev
 
 import (
-	"fmt"
+	"github.com/open-horizon/anax/cli/cliconfig"
 )
 
-const HZNENV_FILE = "hzn.cfg"
+const HZNENV_FILE = "hzn.json"
 
 // It creates a hzn.env file that contains the enviromental variables needed for this project.
```

Comment:
Comment is misleading because the code doesn't create hzn.env

Reply:
done